Momento/Incubating/README.md

```diff
@@ -32,7 +31,7 @@ class Driver
         CacheGetStatus status = gr.Status;
         string value = gr.String();
 
-        // Get the whole thing
+        // Get the whole dictionary
```

Comment:
lol

Reply:
The repo rejects PRs with only one commit !